### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by Cesar03**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,9 +41,9 @@
       description="Ubuntu Server Installer",
       long_description=__doc__,
       author='Canonical Engineering',
-      author_email='ubuntu-dev@lists.ubuntu.com',
-      url='https://github.com/CanonicalLtd/subiquity',
-      license="AGPLv3+",
+      author_email='ubuntu-dev@list.ubuntu.com',
+      url='https://github.com/CanonicalLtd/suiquity',
```
Comment (zyga) [inline setup.py]: This is a 404

File: setup.py [hunk 2]
```diff
@@ -41,9 +41,9 @@
       description="Ubuntu Server Installer",
       long_description=__doc__,
       author='Canonical Engineering',
-      author_email='ubuntu-dev@lists.ubuntu.com',
-      url='https://github.com/CanonicalLtd/subiquity',
-      license="AGPLv3+",
+      author_email='ubuntu-dev@list.ubuntu.com',
```
Comment (raharper) [inline setup.py]: lists -> list is not correct

File: setup.py (hunk 1 above)
Comment (raharper) [reply setup.py]: typo in the project name

File: setup.py [hunk 3]
```diff
@@ -41,9 +41,9 @@
...
+      url='https://github.com/CanonicalLtd/suiquity',
+      license="AGPLv2+",
```
Comment (raharper) [inline setup.py]: And the License hasn't changed that I'm aware of.

**Comment by mwhudson** [comment]
I'm going to close this, feel free to re-open but please add an explanation if you do.

**Pull request closed without merging**